usaspending_api/reporting/v2/views/agencies/toptier_code/overview.py

```diff
@@ -130,7 +130,7 @@ def format_results(self, result_list):
                 "current_total_budget_authority_amount": result["total_budgetary_resources"],
                 "total_budgetary_resources": result["gtas_total_budgetary_resources"],
                 "percent_of_total_budgetary_resources": round(
-                    result["total_budgetary_resources"] * 100 / result["gtas_total_budgetary_resources"], 2
+                    result["total_budgetary_resources"] * 100 / (result["gtas_total_budgetary_resources"] or 100), 2
```

Comment:
If this is suppose to be a percent why are you having the default be to divide by 100?

Reply:
Good question. I put in it to fix divide-by-zero errors and forgot to put in a permanent fix. I think it works because when `gtas_total_budgetary_resources` is null `total_budgetary_resources` is 0.

Reply:
pushed change